assets/js/blocks/product-best-sellers/edit.tsx

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import { useBlockProps } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import { ProductBestSellersBlock } from './block';
+
+export const Edit = ( props: unknown ): JSX.Element => {
```

Comment:
```suggestion
export const Edit = ( props: unknown & Props ): JSX.Element => {
```

In this way, we can avoid a TS error.